### PR TITLE
Document LastCompletionResult isn't available to Scheduled WF runs when coupled with CAN

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

This PR aims to provide clearer documentation around the behavior of the LastCompletionResult when used in conjunction with the ContinueAsNew function within scheduled workflows.

## Notes to reviewers

When a Schedule invokes a WF that successfully completes and returns a result, the Result from the first schedule execution is available to the second scheduled execution via LastCompletionResult

If the during the 2nd run, the WF invokes ContinueAsNew, the LastCompletionResult is NOT available to the new WF run.  

I assume this is the expected behavior, but it is a bit confusing, as the second run isn’t in “Completed” status (it is in “Continued As New” status)

### CODE REVIEW

**Comment by pvsone** [comment]
lgtm

File: docs-src/concepts/what-is-a-schedule.md [hunk 1]
```diff
@@ -183,6 +183,16 @@ And not C, even though C completed after A, because the result for D is captured
 
 Failures and timeouts do not affect the last completion result.
 
+:::note
+
+When a Schedule triggers a Workflow that completes successfully and yields a result, this result from the initial Schedule execution can be accessed by the subsequent scheduled execution through the `LastCompletionResult`.
```
Comment (djmagee) [inline docs-src/concepts/what-is-a-schedule.md:188]:
```suggestion
When a Schedule triggers a Workflow that completes successfully and yields a result, the result from the initial Schedule execution can be accessed by the subsequent scheduled execution through `LastCompletionResult`.
```
Per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.

File: docs-src/concepts/what-is-a-schedule.md [hunk 2]
```diff
@@ -183,6 +183,16 @@ And not C, even though C completed after A, because the result for D is captured
...
+
+It's important to be aware that if, during the second run, the Workflow employs the [Continue-As-New](/concepts/what-is-continue-as-new) function, the `LastCompletionResult` won't be accessible for this new Workflow iteration.
```
Comment (djmagee) [inline docs-src/concepts/what-is-a-schedule.md:190]:
```suggestion
Be aware that if, during the subsequent run, the Workflow employs the [Continue-As-New](/concepts/what-is-continue-as-new) feature, `LastCompletionResult` won't be accessible for this new Workflow iteration.
```

File: docs-src/concepts/what-is-a-schedule.md [hunk 3]
```diff
@@ -183,6 +183,16 @@ And not C, even though C completed after A, because the result for D is captured
...
+
+It is important to note that the [status](/concepts/what-is-a-workflow-execution#status) of the second run is marked as `Continued-As-New`, and not as `Completed`.
```
Comment (djmagee) [inline docs-src/concepts/what-is-a-schedule.md:192]:
```suggestion
It is important to note that the [status](/concepts/what-is-a-workflow-execution#status) of the subsequent run is marked as `Continued-As-New` and not as `Completed`.
```

**Review by axfelix — APPROVED** [review]
LGTM

**Review by flossypurse — APPROVED** [review]

**Pull request merged**